### DISCUSSION TITLE
Schedule an outgoing email

### DISCUSSION THREAD

**Discussion by Taky0nDon** [jeremyephron/simplegmail#106] [Ideas]
I am trying to automate a task that requires sending an email with non-identical content at a certain time. I was hoping this repo would make my attempt to make my life easier, easier. Is this something that could be added to the API? In the meantime I am going to try to accomplish it through the official Gmail API. If I succeed maybe I will fork the repo and try to add the feature. Unless it already exists.

**Comment by jeremyephron** [comment]
Hi @Taky0nDon, I don't believe there is any google API for scheduling emails. You would have to build this by having a program actually send the email at a certain time (this could be done, e.g., by having a cronjob to execute the program which sends the email).

This library provides the tools for interacting with the API, but isn't a service itself which would run a program for you.

**Comment by koliver1137** [comment]
I know this is late, but I would suggest just using crontab and a little bit of python code. There are plenty of tutorials on how to use [Crontab ](https://www.geeksforgeeks.org/crontab-in-linux-with-examples/)if you are a little bit confused with it. And if you already solved this problem and I'm just replying to an old thread for no reason, I hope you have a great rest of your day.

**Comment by jeremyephron** [comment]
As others mentioned, there is no API support for this from Google.

See here for the API feature request: https://issuetracker.google.com/issues/140922183?pli=1.

Closing this issue for now, as it is outside the scope of this package without API support from Google.
